### DISCUSSION TITLE
Search repositories with GitHub Apps

### DISCUSSION THREAD

**Discussion by roi-orca** [community/community#55604] [Apps, API and Webhooks]
When searching for repositories using the search repositories API with an access token of our app, it returns public repositories that weren't installed the app.
It is possible to add some attribute to the search to return only the repositories that match the search but also installed the app? Maybe something like installation:<number> qualifier?

**Comment by JesusCastroFernandez** [comment]
En la API de GitHub, actualmente no existe un calificador específico para buscar repositorios que hayan instalado una aplicación en particular. Los calificadores de búsqueda en la API de repositorios están orientados principalmente a propiedades del repositorio en sí, como el nombre, el lenguaje, la fecha de creación, entre otros.

Sin embargo, hay una forma de obtener una lista de repositorios que hayan instalado tu aplicación utilizando la API de instalaciones (/user/installations o /orgs/{org}/installations) y la API de acceso a repositorios de la instalación (/installation/repositories).

A continuación, te presento un flujo de trabajo general para obtener los repositorios que hayan instalado tu aplicación:

Obtén una lista de instalaciones para tu aplicación utilizando la API de instalaciones (/user/installations o /orgs/{org}/installations). Esto te proporcionará una lista de instalaciones de tu aplicación en cuentas de usuario u organizaciones.

Para cada instalación, puedes obtener una lista de repositorios que están disponibles para esa instalación utilizando la API de acceso a repositorios de la instalación (/installation/repositories). Esto te dará una lista de repositorios específicos que tienen acceso a través de esa instalación.

Con este enfoque, puedes obtener una lista de repositorios que hayan instalado tu aplicación en particular. Luego puedes realizar búsquedas específicas dentro de esa lista utilizando otros calificadores de búsqueda para refinar aún más los resultados.

Recuerda que necesitarás autenticarte y proporcionar los permisos adecuados para acceder a la API de instalaciones y la API de acceso a repositorios de la instalación. Revisa la documentación de la API de GitHub para obtener más detalles sobre cómo realizar estas solicitudes específicas.

**Reply by roi-orca** [reply]
I translated it, I need search capabilities, not only pagination.
I will not get all pages and then filter the results by the search term.
also, there are qualifiers for is:private, is:public, org:, and more, so installation qualifier make sense for me
